hasjob/views/helper.py

```diff
@@ -34,7 +34,7 @@ def sniffle():
         'Cache-Control': 'no-cache, no-store, must-revalidate',
         'Pragma': 'no-cache',
         'Expires': '0'
-        }
+    }
```

Comment:
Undo these indents, please. Set your Flake8 parser to ignore `E501,E128,E123,E124,E402` (this one's E123). 100% PEP8 compliance makes code unreadable.